src/Web/Controller/WordController.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Web\Controller;
+
+use App\Mapper\WordMapper;
+use App\Model\Word;
+use App\Repository\WordRepository;
+use App\Web\Request\Request;
+use App\Web\Response\JsonResponse;
+use App\Web\Response\Response;
+
+class WordController
+{
+    public function __construct(
+        private readonly WordRepository $repo
+    ) {
+
+    }
+
+    public function list(): Response {
+        $words = $this->repo->getWords();
+        $mapper = new WordMapper();
+        $serialized = array_map(fn (Word $item): ?array => $mapper->serialize($item), $words);
+
+        return new JsonResponse(
+            [],
+            $serialized
```

Comment:
Is it necessary to do a double lines?

Also if doing them every line should end with comma. 

Reason: 1. easier to edit 2. does not interfere with git changes if new line needs to be added

Reply:
I forgot to mention so check other places where method/array ends in new line and add commas there. (For example this class constructor could benefit from this)